avaframe/ana3AIMEC/ana3AIMECCfg.ini

```diff
@@ -13,7 +13,9 @@ domainWidth = 600
 # It is possible to force the cell size to take another value, then specify this new value below, otherwise leave empty (default).
 cellSizeSL =
 
-# angle for the start of the run-out zone
+# define a runout area based on an angle of the thalweg profile (requires splitPoint in avaName/Inputs/POINTS as shpFile)
+defineRunoutArea = False
```

Comment:
See comment in python files about maybe using startOfRunoutAngle to achieve the same